.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*       @medic/product-team
```

Comment:
FYI this team includes Derick, Jacqueline, QAs, SREs, etc. https://github.com/orgs/medic/teams/product-team/members
Do you want them signing off on PRs?
Unsure if you want https://github.com/orgs/medic/teams/development-team/members?

Reply:
Yeah I guess. The plan was to work on redefining the teams a little, but development is probably closer to what we want.